rest-api-spec/src/main/resources/rest-api-spec/test/indices.rollover/10_basic.yml

```diff
@@ -76,8 +76,8 @@
 ---
 "Rollover no condition matched":
   - skip:
-        version: "all"
-        reason:  "AwaitsFix - Fails complaining of negative BytesSizeValue"
+        version: " - 5.0.0"
+        reason:  bug fixed in 5.0.1
```

Comment:
Are these the right versions?

Reply:
Yes I think so, these version limits were on the test prior to me muting it, see https://github.com/elastic/elasticsearch/commit/1e9e73ad80282dd84ca1fb290b176645e2952e51

Reply:
Sure, but what about mixed cluster tests where older nodes will be running the buggy code?

Reply:
I think it should be ok (though I'm willing to be proven wrong 😄 ) since the stats are combined on a single node (they are not incrementally added together on different nodes as far as I know), so if this is running on a 5.x node it will run as it did in 5.x (possibly with negative BytesSizeValues but that is allowed on 5.x) and pass. If the test runs against a 6.x node it will have the DocsStats fix to ensure the sizes are added together correctly and the least value passed to the ByteSizeValue will be `-1` which is allowed in 6.x.

Reply:
@jasontedor do you still have concerns here or are you happy for me to merge this PR?

Reply:
Thanks for clarifying.